### PR TITLE
Update README with info on configuring the agent

### PR DESCRIPTION
**Pull request by adjackura**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,14 +5,13 @@ supported Compute Engine [images](https://cloud.google.com/compute/docs/images).
 **Table of Contents**
 
 * [Background](#background)
-* [Agent](#Agent)
+* [Agent](#agent)
```
Comment (illfelder) [inline README.md]: You didn't add a line for "Configuring".

Comment (adjackura) [inline README.md]: Done

File: README.md [hunk 2]
```diff
@@ -29,6 +28,10 @@ Google Compute Engine.
 
 ## Agent
 
+### Configuring
```
Comment (illfelder) [inline README.md]: There should be 4 hash tags instead of 3 to match the other sections.

File: README.md [hunk 3]
```diff
@@ -29,6 +29,10 @@ Google Compute Engine.
...
+
+The agent can be configured either through [metadata or a local config file](https://cloud.google.com/compute/docs/instances/windows/creating-managing-windows-instances#configure-windows-features).
```
Comment (illfelder) [inline README.md]: The agent is configurable through

Comment (illfelder) [inline README.md]: Also, can you move the link onto the next line.

```
The agent is configurable through
[metadata or a local config file](https://cloud.google.com/compute/docs/instances/windows/creating-managing-windows-instances#configure-windows-features).
```

**Pull request merged**